### PR TITLE
[Nim] Extract concepts from protein-translation

### PR DESCRIPTION
**Pull request by ynfle**
I finally got over the "which should I choose to get this 💯 % right" hump and just chose one!!!!

### CODE REVIEW

**Comment by ynfle** [comment]
@reneegyllensvaan I'd love your opinion on this PR.
The idea is to extract the relevant concepts the may have to be mapped, learnt or relearnt when coming from another language

File: languages/nim/reference/exercise-concepts/protein-translation.md
Comment (ErikSchierboom) [inline languages/nim/reference/exercise-concepts/protein-translation.md:34]: This is fine! We don't want to be teaching arithmetic anyways :) You could consider making it more specific by naming it `arithmetic operators`, which _is_ a language concept.

File: languages/nim/reference/exercise-concepts/protein-translation.md
Comment (ErikSchierboom) [inline languages/nim/reference/exercise-concepts/protein-translation.md:4]: I'd include it in this extraction, as we are just extracting what is in the source code without regarding what is being tested for.

File: languages/nim/reference/exercise-concepts/protein-translation.md
Comment (ErikSchierboom) [inline languages/nim/reference/exercise-concepts/protein-translation.md:24]:
```suggestion
- if statement
```

File: languages/nim/reference/exercise-concepts/protein-translation.md
Comment (ynfle) [reply languages/nim/reference/exercise-concepts/protein-translation.md:4]: Done!

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ynfle** [comment]
I think I'd like to wait for the opinion/review of someone that knows the nim side of things before merging.
On the other hand, I don't know if it makes such a difference, being as these aren't such _official_ document.

**Comment by ErikSchierboom** [comment]
@ynfle Feel free to wait, but it doesn't really matter _that_ much, as the concept extractions are just a way to get you used to working with concepts. They'll also serve as the basis for the eventual full Nim concepts list, but that list can do any cleanup and such.

**Comment by ynfle** [comment]
> @ynfle Feel free to wait, but it doesn't really matter _that_ much, as the concept extractions are just a way to get you used to working with concepts. They'll also serve as the basis for the eventual full Nim concepts list, but that list can do any cleanup and such.

Ok 👌

**Pull request merged**
